bin/omarchy-tui-remove

```diff
@@ -6,7 +6,7 @@ DESKTOP_DIR="$HOME/.local/share/applications/"
 if [ "$#" -eq 0 ]; then
   # Find all TUIs
   while IFS= read -r -d '' file; do
-    if grep -q '^Exec=.*alacritty.*-e' "$file"; then
+    if grep -q '^Exec=.*$TERMINAL.*-e' "$file"; then
```

Comment:
This looks like a bug, `$TERMINAL` will not be expanded in single quotes.

Reply:
def a bug, please pr

Reply:
When looking closer it seems like it is not a bug after all, seems like the intentions is actually to grep for the string `$TERMINAL` unexpanded.